apps/roam/src/utils/createReifiedBlock.ts

```diff
@@ -117,15 +112,12 @@ export const createReifiedRelation = async ({
   relationBlockUid: string;
   destinationUid: string;
 }): Promise<string | undefined> => {
-  const authorized = getSetting("use-reified-relations");
-  if (authorized) {
-    return await createReifiedBlock({
-      destinationBlockUid: await getOrCreateRelationPageUid(),
-      schemaUid: relationBlockUid,
-      parameterUids: {
-        sourceUid,
-        destinationUid,
-      },
-    });
-  }
+  return await createReifiedBlock({
```

Comment:
nit `createReifiedRelation` is now only used to call another function, so it should probably be removed.

Reply:
`createReifiedBlock` is the generic function that I expect to keep using. The other one is syntactic sugar for a special case, even if it's the only case being used now.

Reply:
If it is just syntactic sugar it can be renamed on import.  It doesn't need a separate function.